pinot-common/src/main/java/com/linkedin/pinot/common/request/helper/ControllerRequestBuilder.java

```diff
@@ -157,6 +170,9 @@ public static JSONObject buildCreateRealtimeTableJSON(String tableName, String s
     if (noDictionaryColumns != null) {
       tableIndexConfig.put("noDictionaryColumns", noDictionaryColumns);
     }
+    if (partitioners != null) {
```

Comment:
Since we are passing empty map in one of the calls, it is going to write out an empty config? I'd rather have nothing written at all in that case.